libclambcc/ClamBCExtendPHIsTo64Bit/ClamBCExtendPHIsTo64Bit.cpp

```diff
@@ -135,10 +140,14 @@ class ClamBCExtendPHIsTo64Bit : public ModulePass
             convertPHIs(pFunc);
         }
 
-        return true;
+        if (bChanged){
```

Comment:
It would be nice to have a comment here to explain the `none()` vs. `all()`

Reply:
Done.